chapter1/configuration.md

```diff
@@ -1,4 +1,55 @@
-- Webportal login
-- navigators, options
-  (Screen shots)
+## Basic Configuration
+
+### Launch webportal
+In `appliance_console`, we need to check either `ManageIQ Server` is up. If not then we can start it by entering digit `14` in the advanced setting.  After that when we hit IPv4 address which is given in `appliance_console` in web browser we can see the ManageIQ's Login page.
+
+![login](../images/chapter1/manageiq_login.png "ManageIQ Log In Page")
+
+### Log In
+There are default credentials for ManageIQ. We can change it but we will discuss it later.
+
+- **Login:** admin
+- **Password:** smartvm
+
+Now we can see the dashboard which looks like following image
+
+![dashboard](../images/chapter1/manageiq_dashboard.png "ManageIQ Dashboard")
+
+In the extreme left, we can see the vertical navigation bar. We can easily navigate through elements from menus of this bar.
+
+### Change Configuration
+In upper right corner, we can see dropdown bar for EVM. Go to **EVM  &rarr;  Configuration**
```

Comment:
write as  `we can see dropdown menu for EVM.`

Reply:
Done